package.json

```diff
@@ -1,22 +1,18 @@
 {
   "name": "openfin-service-tooling",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "",
-  "main": "dist/index.js",
+  "main": "index.js",
   "scripts": {
-    "check": "gts check",
-    "clean": "gts clean",
-    "fix": "gts fix && cd demo && gts fix",
-    "build": "tsc -p tsconfig.json",
-    "prepack": "npm run clean && npm run build",
+    "check": "eslint src demo --ext .ts --ext .tsx --config src/config/.eslintrc.json",
+    "clean": "rimraf dist",
+    "fix": "eslint src demo --ext .ts --ext .tsx --config src/config/.eslintrc.json --fix",
+    "build": "npm run clean && tsc -p tsconfig.json && node src/fileCopy.js",
     "demo": "npm run build && cd demo && npm run start",
     "demo:build": "npm run build && cd demo && npm run build",
-    "start": "npm run build && node dist/cli.js start"
+    "start": "npm run build && node dist/cli.js start",
+    "deploy:test": "npm run build && cd dist && npm pack"
```

Comment:
Maybe name this `demo:pack`, to align with the `demo` and `demo:build` scripts?..

It doesn't "deploy" anything, and "test" in a script kinda implies something more like unit testing (to me, at least).

Reply:
This was a local test - I have removed it now.